store/store.go

```diff
@@ -683,13 +683,13 @@ func (s *Store) Apply(l *raft.Log) interface{} {
 	}
 }
 
-// Snapshot returns a snapshot of the database. The caller must ensure that
+// RawDBSnapshot returns a snapshot of the database. The caller must ensure that
 // no transaction is taking place during this call. Hashicorp Raft guarantees
```

Comment:
Right, the lack of a leader check is intentional. We want the current state of the node's database as-is.